prisma/schema.prisma

```diff
@@ -188,9 +189,21 @@ model Exercise {
   flagReason  String?
   flaggedAt   DateTime?
   flaggedById Int?
-  author      User      @relation(fields: [authorId], references: [id], onDelete: Cascade)
-  flaggedBy   User?     @relation("flaggedExercises", fields: [flaggedById], references: [id])
-  module      Module    @relation(fields: [moduleId], references: [id], onDelete: Cascade)
+  author      User                 @relation(fields: [authorId], references: [id], onDelete: Cascade)
+  flaggedBy   User?                @relation("flaggedExercises", fields: [flaggedById], references: [id])
+  module      Module               @relation(fields: [moduleId], references: [id], onDelete: Cascade)
+  submissions ExerciseSubmission[]
 
   @@map("exercises")
 }
+
+model ExerciseSubmission {
```

Comment:
I see some benefits for `submittedAt`. But it's one of the things that is more suitable to be added once needed.

Reply:
Yeah, I agree, we can add fields when we need them